code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -777,8 +777,7 @@
 	fire_delay = 1
 	burst_size = 1
 	spread = 1
-	extra_damage = 30
-	extra_penetration = 0.1
```

Comment:
I suggest either removing the AP or keeping the damage 30 or reverting the damage to it's original version and keeping the AP